trl/trainer/sft_trainer.py

```diff
@@ -250,7 +251,7 @@ def make_inputs_require_grad(module, input, output):
         if formatting_func is None and dataset_text_field is None:
             # check if dataset has ChatML format or instruction format and is supported
             # if not stays #None
-            formatting_func = get_formatting_func_from_dataset(train_dataset, tokenizer)
+            formatting_func = get_formatting_func_from_dataset(train_dataset, tokenizer, apply_chat_instruction_template)
 
         requires_input_output_keys = False
```

Comment:
This variable name should probably be changed - it doesn't make sense if the keys are prompt / completion

Reply:
ya I changed it below and left that one place, thanks for catching it